data/test/news/articles/c0000000017o.json

```diff
@@ -188,12 +188,12 @@
                     {
                       "type": "paragraph",
                       "model": {
-                        "text": "Maecenas varius diam in mi dignissim, eget maximus lacus commodo",
+                        "text": "Maecenas $,%,+,-,=,_,£,¥,©,°,²,³,¹,×,ẞ,ß,à,á,â,æ,ç,è,é,ê,î,ô,÷,û,Ŵ,ŵ,Ŷ,ŷ,⁰,⁴,⁵,⁶,⁷,⁸,⁹,₀,₁,₂,₃,₄,₅,₆,₇,₈,₉,€ diam in mi dignissim, eget maximus lacus commodo",
```

Comment:
Just double checking, do the characters come from aries like this, or are they encoded in any way?

Reply:
Good question 👍don't know! Will ask in Slack

Reply:
I tried to make test asset cww18elgprxo on test to check, but it never surfaced 🙈

Reply:
Cameron on slack confirmed they are not encoded, so all good